### ISSUE TITLE
Some custom ops with weird chars fail to work

### ISSUE THREAD

**Issue by zoffixznet** [rakudo/rakudo#1264]
With some weird Unicode chars for custom operators, the definition succeeds and installs the op, but trying to use it fails.

I think these should either work, or the definition should complain about invalid character.

```bash
$ perl6 -v
This is Rakudo version 2017.11-2-g575d31e built on MoarVM version 2017.11
implementing Perl 6.c.

$ perl6 -e 'sub infix:«"z"» ($,$) { say "here" }; "42 z 42".EVAL'
here

$ perl6 -e 'sub infix:«"\x[2028]"» ($,$) { say "here" }; "42 \x[2028] 42".EVAL'
===SORRY!=== Error while compiling /home/zoffix/EVAL_0
Confused
at /home/zoffix/EVAL_0:2
------> 42⏏ 
    expecting any of:
        infix
        infix stopper
        statement end
        statement modifier
        statement modifier loop
```

**Comment by zoffixznet** [comment]
Actually, the problematic char matches `\s` and should prolly get same error as trying to use any other whitespace:

```
$ perl6 -e 'say "\x[2028]" ~~ /\s/'
｢ ｣
```

**Comment by zoffixznet** [comment]
tl;dr DIHWIDT

Never mind. I was misremembering the error. If you try to use whitespace around the edges of the op, it just ignores it. Inside, it [just splits on space](https://github.com/rakudo/rakudo/blob/0a24efc32d12500d6973c50162d82952db431455/src/Perl6/Grammar.nqp#L4890), trying to figure out how many parts are there and so potential results are "Null operator" death or "Too many parts" death.

The `\x[2028]` slips through that split and gets installed and I'm guessing later conflicts with `<ws>`.
